src/protocol-parser/protocol-parser.ts

```diff
@@ -270,24 +270,27 @@ export namespace Script {
   //   ?maxObjectDepth: (js-uint / null) .default null,
   //   ?includeShadowTree: ("none" / "open" / "all") .default "none",
   // }
-  const SerializationOptions = zod.object({
-    maxDomDepth: zod.number().int().min(0).optional(),
-    maxObjectDepth: zod.number().int().min(0).max(MAX_INT).optional(),
+  const SerializationOptionsSchema = zod.object({
+    maxDomDepth: zod.union([zod.null(), zod.number().int().min(0)]).optional(),
+    maxObjectDepth: zod
+      .union([zod.null(), zod.number().int().min(0).max(MAX_INT)])
+      .optional(),
     includeShadowTree: zod.enum(['none', 'open', 'all']).optional(),
```

Comment:
Unrelated to this PR, but if you add `.default()` (which I do recommend!), ensure to also add it here, set to `'none'`.

Reply:
https://github.com/GoogleChromeLabs/chromium-bidi/pull/726#discussion_r1195130999